pkg/audit/manager.go

```diff
@@ -248,6 +251,10 @@ func (am *Manager) auditResources(ctx context.Context) ([]*constraintTypes.Resul
 			}
 
 			for _, obj := range objList.Items {
+				if am.configMatchSet.ExcludedNamespaces[match.Audit][obj.GetNamespace()] {
```

Comment:
This needs a locking primitive for thread safety.

Reply:
added